src/Microsoft.TemplateEngine.Cli/TableOutput/TemplateGroupDisplay.cs

```diff
@@ -24,8 +29,13 @@ internal static class TemplateGroupDisplay
         /// <param name="templateList">list of templates to be displayed.</param>
         /// <param name="language">language from the command input.</param>
         /// <param name="defaultLanguage">default language.</param>
+        /// <param name="environment"><see cref="IEnvironment"/> settings to use.</param>
         /// <returns></returns>
-        internal static IReadOnlyList<TemplateGroupTableRow> GetTemplateGroupsForListDisplay(IEnumerable<ITemplateInfo> templateList, string? language, string? defaultLanguage)
+        internal static IReadOnlyList<TemplateGroupTableRow> GetTemplateGroupsForListDisplay(
```

Comment:
nit: Wondering if it would make more sense to go with `IEngineEnvironmentSettings` or `string newLine`.

Reply:
I think making class not static with settings stored may make it difference, but for now will keep as is - whatever we pass, we need to pass it most methods.